pacejka_model:
  C_Pf_model: [7.173657737630526, 1.5628207135320944, 0.3882069183399687, 0.5278435112876324]
  C_Pr_model: [8.288516161529747, 2.109127387401536, 0.3751749393933447, 0.4036016891841555]

pacejka_ref:
  C_Pf_ref: [7.17, 1.56, 0.69, 0.53]
  C_Pr_ref: [8.29, 2.11, 0.68, 0.40]
